src/utils/common.util.ts

```diff
@@ -1,14 +1,15 @@
 /**
  * @returns Array of Object keys with their proper types. Use this instead of Object.keys
  */
-export function typedKeys<O extends object, K extends keyof O = keyof O>(
-    obj: O
-  ): K[] {
-    return Object.keys(obj) as K[]
-  }
+export function typedKeys<O extends object, K extends keyof O = keyof O>(obj: O): K[] {
+  return Object.keys(obj) as K[];
+}
 
+export function TypedKeys<TObj extends object>(obj: TObj) {
+  return Object.keys(obj) as (keyof TObj)[];
+}
```

Comment:
@OswaldUnity Which function should I use?

Reply:
Second one is best one, but we shouldn't be using that, if there is a need for those functions means we are using objects without types